src/modules/VirtConfig.rb

```diff
@@ -317,6 +318,70 @@ def LXCDialog
         )
       end
 
+      Builtins.y2milestone("VirtConfig::ConfigureDom0: Checking for Installed Patterns and Packages")
+      is_xen_server_installed = false
+      is_xen_tools_installed = false
+      is_kvm_server_installed = false
+      is_kvm_tools_installed = false
+      is_client_tools_installed = false
+      if isOpenSuse == true
+        # On openSUSE there are no 'tools' patterns for Xen and KVM
+        if Package.Installed("patterns-openSUSE-xen_server")
```

Comment:
Again, this is a boolean value, simple `is_xen_server_installed = Package.Installed("patterns-openSUSE-xen_server")` can be used instead of the whole condition.

Similarly for the other package tests.